test/README.md

```diff
@@ -20,297 +20,130 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# How-to Run Tests
-Prerequisites:
+# Quick Start
+```
+    cd openj9/test/TestConfig
+    export JAVA_BIN=/my/openj9/sdk/bin
+    export SPEC=linux_x86-64_cmprssptrs
+    make -f run_configure.mk   // generates makefiles
+    make compile               // downloads test related material/libs and compiles test material 
+    make _sanity.functional    // runs tests
+```
+
+Please read [OpenJ9 Test User Guide](./docs/OpenJ9TestUserGuide.md) for details.
 
-  * ant 1.7.1 or above with ant-contrib.jar
-  * make 3.81 or above (recommend make 4.1 or above on windows)
-  * perl 5.10.1 or above with JSON and Text::CSV module installed
-  * wget 1.14 or above
+## Prerequisites:
+Please read [Prerequisites.md](./docs/Prerequisites.md) for details.
 
-Let's create the following scenario:
-For the purpose of this example, we assume you have an OpenJ9 SDK ready
-for testing. Below are the specific commands you'd run to compile and
-run tests. Details are explained in *Tasks in OpenJ9 Test* section below.
+# Common User Cases
+## 1) Execute a directory of tests only
 
 ```
-cd openj9/test/TestConfig
-export JAVA_BIN=<path to JDK bin directory that you wish to test>
-export SPEC=linux_x86-64_cmprssptrs
-make -f run_configure.mk
-make test
+    make -C ../JIT_Test -f autoGen.mk _sanity
```

Comment:
Maybe you also need to mention the other way:
`cd ../JIT_Test
make -f autoGen.mk _sanity`